lib/aggregate/aggregate_store.rb

```diff
@@ -178,6 +208,10 @@ def aggregate_attribute_changed?(agg_attribute)
       aggregate_changes[agg_attribute.name] || Array.wrap(aggregate_values[agg_attribute.name]).any? { |value| value.try(:changed?) }
     end
 
+    def aggregate_attribute_saved_changed?(agg_attribute)
+      Array.wrap(aggregate_values[agg_attribute.name]).any? { |value| value.try(:saved_changes?) }
```

Comment:
This is helpful for looking at attributes that are themselves PORO Aggregate Attributes, but I think we also want to check `aggregate_attribute_saved_changes` for the attribute name like we do above in `aggregate_attribute_changed?`, right?